### DISCUSSION TITLE
Flutter App Errors building iOS: app_settings module not found.

### DISCUSSION THREAD

**Discussion by ErinnPsajdPowertrust** [codemagic-ci-cd/codemagic-docs#1860] [Q&A]
-  I confirm I have searched the [Docs](https://docs.codemagic.io), [Codemagic Sample Projects](https://github.com/codemagic-ci-cd/codemagic-sample-projects), and GitHub [Discussions](https://github.com/orgs/codemagic-ci-cd/discussions).
-  I confirm the software versions on my local computer match the ones in CI environment (e.g. Xcode version, Gradle version). See [here](https://docs.codemagic.io/troubleshooting/common-android-issues/#builds-work-locally-but-fail-on-codemagic)
- I confirm I am able to do a fresh clone of the repository and build the project on my local computer.

### Which mobile framework are you using?

Flutter (Dart)

### Steps to reproduce

An earlier issue seemed to be with cocoapods, so I manually added Podfile to the ios folder containing: 

platform :ios, ’10.0’
target 'Runner' do
  use_frameworks!
end

### Expected results

Expected Codemagic uploading app to Testflight.

### Actual results

The error Modile 'app_settings' not found is displaying in the log. A snippet of the log below: 

Running pod install...                                             682ms
Running Xcode build...                                          
Xcode archive done.                                         50.3s
Failed to build iOS app
Parse Issue (Xcode): Module 'app_settings' not found
/Users/builder/clone/ios/Runner/GeneratedPluginRegistrant.m:11:8

Encountered error while archiving for device.

Build failed :|
Failed to build for iOS

**Comment by himesh-cm** [comment]
Hi @ErinnPsajdPowertrust, Thanks for sharing the info, can you please share the build id as well?

**Comment by himesh-cm** [comment]
Hi @ErinnPsajdPowertrust, I'm tentatively closing the discussion since we haven't heard back from you. You can open the discussion again by replying to this message or opening a new discussion.
